core/src/main/scala/filodb.core/memstore/PartKeyLuceneIndex.scala

```diff
@@ -146,7 +157,7 @@ class PartKeyLuceneIndex(ref: DatasetRef,
 
     def addField(name: String, value: String): Unit = {
       // Use PartKeyIndexBenchmark to measure indexing performance before changing this
-      if (facetEnabled && name.nonEmpty && value.nonEmpty) {
+      if (facetEnabledForLabel(name) && name.nonEmpty && value.nonEmpty) {
```

Comment:
Nit pick, but by adding ``facetEnabledForLabel`` we can prevent it from being invoked in case of empty name or value

Reply:
done